### ISSUE TITLE
compute-saddle problem

### ISSUE THREAD

**Issue by dongmei-wang** [open2c/cooltools#248]
Hello,

I got the following error when running compute-saddle, and have no idea what could have caused the error:
File "/opt/anaconda3/envs/cooltools/lib/python3.9/site-packages/cooltools/cli/compute_saddle.py", line 393, in compute_saddle
    if len(range_):
TypeError: object of type 'NoneType' has no len()

My command is this: cooltools compute-saddle -t cis --qrange 0.02 0.98 --strength -o red_blood_Z-VAD_100K.saddle --fig pdf red_blood-Arima-R1-filtered.mcool::resolutions/100000 red_blood_100K.cis.vecs.tsv::E1 red_blood_expected.tsv::balanced.avg

Thank you,
Dongmei

**Comment by nvictus** [comment]
If you are using `--qrange` you need to specify `--quantiles`.

**Comment by dongmei-wang** [comment]
Thank you for your response. I gave my files to my colleague and he ran successfully with the same command. I was using cooltools v0.4.0, he was using v0.3.2, so I downgraded my cooltools to v0.3.2, now I got a different error: 
File "/opt/anaconda3/envs/cooltools/lib/python3.9/site-packages/pandas/io/parsers.py", line 1162, in _validate_usecols_names
    raise ValueError(
ValueError: Usecols do not match columns, columns expected but not found: ['chrom']

I think it may have something to do with my environment set up for cooltools, even though I was able to run call-compartments and compute-expected successfully. 

I am new to python as well as HiC analysis, I may have made some silly mistakes I don't know.

Thank you,
Dongmei

**Comment by Phlya** [comment]
This sounds like your expected file was generated using the new version - either rename the first column to "chrom" instead of "region", or re-make it with 0.3.2.

But we need to sort out what's not working in 0.4 then...

**Comment by dongmei-wang** [comment]
I changed "region" to "chrom", and got the previous error message back. So it may not be a version related problem.

I used conda virtual env to install all packages, could this cause any package incompatibility problem?

Thank you for your help!
Dongmei

**Comment by dongmei-wang** [comment]
I just got it to work!

I had cooltools installed under general environment first, but when I install it, it gave me an error, has something to do with some package version is not compatible. So I changed to use conda environment to re-install everything. The error message came from using cooltools installed under conda environment. When I tried without activate conda env, it worked. So it does have something to do with conda install. Even though I don't know what it is.

Thank you,
Dongmei
